### DISCUSSION TITLE
Is it possible to make the tray icon changeable?

### DISCUSSION THREAD

**Discussion by Ghost217** [diospiroverde/WazzApp#3] [Feature Requests]
Hello diospiroverde, first of all thank you very much for your work and the app, it works great. Is it possible to make the tray icon changeable? I achieved it by replacing the 4 icons (https://github.com/diospiroverde/WazzApp/tree/main/src/icon) locally with my own images and rebuild the app. Alternatively, a choice between a momochomous and colored tray icon would be great. Best regards

**Comment by diospiroverde** [comment]
HI, Ghost217.

Thank you very much for your feedback. That's a very good idea, and I will add this option in the next release.

All the best.

**Comment by diospiroverde** [comment]
Hi.

I've added the option for a monochrome or colored Tray Icon (colored being default). In the next release I'll add the option to choose a custom Tray Icon from a file along with other options. If you have any ideas about design or anything else, please share.

All the best.

**Reply by Ghost217** [reply]
I just took the WhatsApp logo (https://www.internetmatters.org/wp-content/uploads/2018/02/Whatsapp-logo-pc-600x314.png). I think that's OK for private use. I don't know what to consider if you want to publish it. For me, your blue icon is completely fine and looks great. Thank you :)
